layout/container.js

```diff
@@ -18,4 +20,4 @@ class LayoutContainer extends React.Component {
   }
 }
 
-module.exports = LayoutContainer
+module.exports = DragDropContext(HTML5Backend)(LayoutContainer)
```

Comment:
is this the right place to add? needs to 'wrap your app component'. this is the highest level place i could find for that, other than routes.js